nussl/separation/duet.py

```diff
@@ -388,7 +504,10 @@ def plot(self, output_name, three_d_plot=False, normalize=True):
         AA = np.tile(self.attenuation_grid[1::], (self.num_delay_bins, 1)).T
         DD = np.tile(self.delay_grid[1::].T, (self.num_attenuation_bins, 1))
 
-        histogram_data = self.attenuation_delay_histogram if normalize else self.smoothed_hist
+        # save smoothed_hist (not normalized) as an option for plotting
+        smoothed_hist = self._smooth_matrix(self.non_normalized_hist, np.array([3]))
```

Comment:
Where does this `np.array([3])` come from? You should change `self._smooth_matrix` so that it doesn't take a numpy array of a single element. (Unless it needs to, in which case you need to justify that and explain what's going on here.) Also, why 3?